app/controllers/alaveteli_pro/draft_info_requests_controller.rb

```diff
@@ -0,0 +1,121 @@
+# -*- encoding : utf-8 -*-
+# app/controllers/alaveteli_pro/request_controller.rb
+# Dashboard controller, for pro user dashboards.
+#
+# Copyright (c) 2008 UK Citizens Online Democracy. All rights reserved.
+# Email: hello@mysociety.org; WWW: http://www.mysociety.org/
+
+class AlaveteliPro::DraftInfoRequestsController < AlaveteliPro::BaseController
+  def new
+    @draft_info_request = DraftInfoRequest.new
+  end
+
+  def edit
+    @draft_info_request = DraftInfoRequest.find(params[:id])
+  end
+
+  def create
+    @draft_info_request = DraftInfoRequest.new(draft_info_request_params)
+    @draft_info_request.user = current_user
+    if @draft_info_request.save
+      redirect_to action: :show, id: @draft_info_request.id
+    else
+      render "new"
+    end
+  end
+
+  def update
+    @draft_info_request = DraftInfoRequest.find(params[:id])
+    @draft_info_request.update_attributes(draft_info_request_params)
+    if @draft_info_request.save
+      redirect_to action: :show, id: @draft_info_request.id
+    else
+      render "new"
+    end
+  end
+
+  def show
+    @draft_info_request = DraftInfoRequest.find(params[:id])
+  end
+
+  def publish
```

Comment:
I think we should try to stick to the default RESTful actions for new controllers.

I'd make this something like `DraftPublicationsController#create`.